app.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const data = require('./db/data/test-data/index.js')
+const {getAllTopics} = require('./Controllers/controller.js')
+
+const app = express()
+
+app.use(express.json())
```

Comment:
We don't need this line of code yet. We will need it once we make a post. Please remove.